h/textutil.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
Good call. I evaluated a couple which all sucked for unicode, but I missed this one.